Request translators to update the CODEOWNERS file (#7798)
ghostty-org/ghostty: po/README_TRANSLATORS.md

You should then be able to run `zig build` and see your translations in action.

Before opening a pull request with the new translation file, you should also add
the locale to the `CODEOWNERS` file. Find the `# Localization` section near the

> bottom and add a line like so:
> 
> ```diff
>  # Localization
>  /po/README_TRANSLATORS.md @ghostty-org/localization
>  /po/com.mitchellh.ghostty.pot @ghostty-org/localization
>  /po/zh_CN.UTF-8.po @ghostty-org/zh_CN
> +/po/xx_YY.UTF-8.po @ghostty-org/xx_YY
> ```
> 
> Where `xx_YY` would be your locale.
> 

Reviewer: I think the last sentence looks a bit lonely/awkward there.
`````suggestion
bottom and add a line like so (where `xx_YY` is your locale):

```diff
 # Localization
 /po/README_TRANSLATORS.md @ghostty-org/localization
 /po/com.mitchellh.ghostty.pot @ghostty-org/localization
 /po/zh_CN.UTF-8.po @ghostty-org/zh_CN
+/po/xx_YY.UTF-8.po @ghostty-org/xx_YY
```
`````
